apps/ci/ci-conf.sh

```diff
@@ -59,6 +59,12 @@ case $COMPILER in
     echo "CCOMPILERCXX=\"clang++-9\"" >> ./conf/config.sh
     ;;
 
+  "clang8" )
+    time sudo apt-get install -y clang-8
+    echo "CCOMPILERC=\"clang-8\"" >> ./conf/config.sh
+    echo "CCOMPILERCXX=\"clang++-8\"" >> ./conf/config.sh
+    ;;
+
```

Comment:
we probably should put clang8 before clang9

Reply:
ah definitely yes, my bad